website/content/en/docs/concepts/_index.md

```diff
@@ -0,0 +1,64 @@
+
+---
+title: "Intro to Karpenter Concepts"
+linkTitle: "Concepts"
+weight: 20
+resources:
+- src: "**fig1*.png"
+- src: "**fig2*.png"
+---
+
+Kubernetes is designed to schedule new pods onto existing nodes. However, when existing nodes get full, Kubernetes struggles with automatically provisioning new nodes from a cloud provider.
+
+Karpenter activates when pending pods are unable to be fit onto existing nodes. Karpenter understands the instance types available from cloud providers, such as AWS. Karpenter’s provisioner intelligently selects new instances to start, using rich data from both podspecs and the cloud provider APIs (such as EC2’s flexible fleet API). 
```

Comment:
I'd avoid language like "understands", "intelligently" and "rich data". These words don't mean anything in practice.

Reply:
I'd like to get @akestner 's perspective on this. I think there is a difference between a marketing ("buy aws services") and highlighting the advantages of karpenter ("how our design and approach is different from CAS").